docs/myriad-remote-distribution-configuration.md

```diff
@@ -0,0 +1,47 @@
+#Myriad Remote Distribution
+
+The Myriad Scheduler can be configured to automatically download and run the hadoop yarn binaries. This
+means you won't have to install and configure hadoop yarn on each machine.  However, this comes at the cost of
+downloading the binaries each time a node manager is spawned.  One should consider this tradeoff carefully.  Note
+this is a very new feature and the configuration options may change dramatically in the future.
+
+# Myriad Remote Distribution Bundle Creation
+
+We will assume you are using hadoop-2.5.0 downloaded from hadoop.apache.org.  Specific vendor versions should 
+work but may require additional steps.  We will also assume use installed hadoop in /opt/hadoop-2.5.0, adjust
+this path to fit your installation.
+
+First configure the Resource Manager as normal.
+
+In the myriad folder edit src/main/resources/myriad-config-default.  For standard configuration see
```

Comment:
Use backticks (\`) in markdown to surround `paths`, `filenames`, and `classes/variables`, so they look different from normal text.